app/src/main/java/com/architect/coders/mu8/repository/IComicRepository.kt

```diff
@@ -0,0 +1,9 @@
+package com.architect.coders.mu8.repository
+
+import com.architect.coders.mu8.TS
+import com.architect.coders.mu8.remote.model.getallComic.GetAllComicsDTO
+import retrofit2.Response
+
+open interface IComicRepository {
+    suspend fun getAllComics( ts: String = TS): Response<GetAllComicsDTO>
```

Comment:
Remove the space before the paremeter and rename `ts` for `timestamp`, avoid the usage of short name variables that doesn't tell exactly what it is in an obvious way.